src/index.js.flow

```diff
@@ -113,7 +113,7 @@ declare export function createChannel(connection: AmqpConnection|Promise<AmqpCon
 
 declare export function createManagedChannel(connection: AmqpManagedConnection): PublishChannel
 
-declare export function consumeFrom<C:ConsumeChannel> (channel: C): (config: ConsumeConfig, messageHandler: MessageHandler) => Promise<{}>
+declare export function consumeFrom<C:DuplexChannel> (channel: C): (config: ConsumeConfig, messageHandler: MessageHandler) => Promise<{}>
```

Comment:
Take email-consumer as an example:
On [line 42](https://github.com/nowaitapp/payments/pull/287/files#diff-6c8ae6129c5aee8ed03cabbcbb89dfafL42), `amqplib.connect` return [`Promise<AmqpConnection>`](https://github.com/nowaitapp/payments/pull/287/files#diff-502fb908a94f81fb775adf2fdd36db0cR10) 
And `createChannel(connection)` return [`AmqpDuplexChannel`](https://github.com/nowaitapp/payments/pull/287/files#diff-502fb908a94f81fb775adf2fdd36db0cR6) which does not match the original type `ConsumeChannel `